.github/pull_request_template.md

```diff
@@ -1,68 +1,33 @@
-<!--
+## 🎫 Ticket
 
---- PR Hygiene Checklist ---
+https://jira.cms.gov/browse/BCDA-xxx
```

Comment:
May need to switch back to "Fixes BCDA-xxx" if the Jira integration doesn't pick this up. Will wait a little while to see if the PR shows under the Development section.

Reply:
It works as is! yay!